lib/hoe.rb

```diff
@@ -631,6 +645,7 @@ def initialize name, version = nil # :nodoc:
     self.extra_dev_deps       = []
     self.extra_rdoc_files     = []
     self.licenses             = []
+    self.metadata             = {}
```

Comment:
Sorry. Catch me up. What purpose does this serve?

Reply:
An accident from other approaches. Removed now.